### PR TITLE
Announcements import/export during workspace import/export

### PR DESCRIPTION
**Pull request by kitan1982**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Fixed tickets | #712 partially

### CODE REVIEW

File: plugin/announcement/Transfer/AnnouncementImporter.php [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+                            ->end()
+                        ->end()
+                    ->end()
+                ->end()
+            ->end();
+    }
+
+    public function supports($type)
+    {
+        return $type === 'yml' ? true : false;
```
Comment (stefk) [inline plugin/announcement/Transfer/AnnouncementImporter.php]: Just: `return $type === 'yml'`

File: plugin/announcement/Transfer/AnnouncementImporter.php [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+    }
+
+    public static function fileNotExists($v, $rootpath)
+    {
+        $ds = DIRECTORY_SEPARATOR;
+
+        return !file_exists($rootpath.$ds.$v);
+    }
+
+    public function import(array $datas)
```
Comment (stefk) [inline plugin/announcement/Transfer/AnnouncementImporter.php]: `$data` is always singular.

File: plugin/announcement/Transfer/AnnouncementImporter.php [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+                $announcement->setContent($content);
+                $this->om->persist($announcement);
+            }
+        }
+        $this->om->persist($announcementAggregate);
+
+        return $announcementAggregate;
+    }
+
+    public function export(Workspace $workspace, array &$_files, $object)
```
Comment (stefk) [inline plugin/announcement/Transfer/AnnouncementImporter.php]: The underscore in `$_files` is against coding conventions.

**Comment by stefk** [comment]
LGTM

**Comment by Elorfin** [comment]
LGTM

**Pull request merged**